### PR TITLE
Silence yacc warnings

### PR DESCRIPTION
**Pull request by Algunenano**
I was getting tired of them.

Fixes https://trac.osgeo.org/postgis/ticket/4473 but it

### CODE REVIEW

File: liblwgeom/lwin_wkt_parse.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.4.  */
+/* A Bison parser, made by GNU Bison 3.4.2.  */
```
Comment (Komzpa) [inline liblwgeom/lwin_wkt_parse.h]: Can we do something about not committing this file into repo? It looks completely generated, and seems to be going to be built anyway during build process.

Comment (strk) [reply liblwgeom/lwin_wkt_parse.h]: Unexpectedly I've been asking for this for a long time now. Contrary argument has always been: not everyone can easily get a parser generator nowadays ... Please please let's get rid of it now!

Comment (Algunenano) [reply liblwgeom/lwin_wkt_parse.h]: I'd be ok with it. We just need to make sure it can also be generated with the tarball releases.

**Comment by strk** [comment]
> I'd be ok with it. We just need to make sure it can also be generated with the tarball releases.

Yes, make_dist.sh would need a tweak to fix that.
The "distclean" rule should be already NOT removing that file.

**Comment by Algunenano** [comment]
I've created a different issue to handle the file removal (https://trac.osgeo.org/postgis/ticket/4584)

**Pull request closed without merging**